src/Digdir.Domain.Dialogporten.WebApi/Endpoints/V1/ServiceOwner/ServiceOwnerGroup.cs

```diff
@@ -4,7 +4,7 @@ namespace Digdir.Domain.Dialogporten.WebApi.Endpoints.V1.ServiceOwner;
 
 public sealed class ServiceOwnerGroup : Group
 {
-    public const string RoutePrefix = "Serviceowner";
+    public const string RoutePrefix = "serviceowner";
```

Comment:
```suggestion
    public const string RoutePrefix = "Serviceowner";
```

Think we have to let this one stay capitalized, lest our ugly duckling patch gets its own category in the GUI.

![CleanShot 2025-02-21 at 10 28 24@2x](https://github.com/user-attachments/assets/299e654a-9156-4632-be1c-a16bbaeadd2b)

Reply:
Argh, should have realized it was a reason for this. Will need to fix the comparison in SecurityRequirementsOperationProcessor as well.

Reply:
Or change "enduser" to "Enduser", which would at least make it consistent.

Reply:
Or, change the patch controller to have a magic string capitalized 🤷🏼